### DISCUSSION TITLE
I did not get any points of gssoc

### DISCUSSION THREAD

**Discussion by VishvamPorwal** [vikasdo/Book-Recommendation-Analysis#56] [General]
I resolved 4 issues labelled gssoc21 with level as well https://gssoc.girlscript.tech/profile.html?id=VishvamPorwal
can you look into the matter please
@vikasdo @Akshay6890 
-Thanks.

**Comment by VishvamPorwal** [comment]
Sir, I looked around and I found out that you are supposed to add labels to pr's as well.
Please do the same.

**Comment by vikasdo** [comment]
all have been resolved please check

**Reply by VishvamPorwal** [reply]
Yeah, thanks.
